flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/WindowedStream.java

```diff
@@ -800,4 +907,39 @@ public StreamExecutionEnvironment getExecutionEnvironment() {
 	public TypeInformation<T> getInputType() {
 		return input.getType();
 	}
+
+	private static <T, R, K, W extends Window> ProcessWindowFunction<T, R, K, W> wrapWindowFunction(final WindowFunction<T, R, K, W> cleanedFunction) {
```

Comment:
There is already `WrappingFunction`, that can be used for these kinds of purposes.